lib/twitter-ads/audiences/tailored_audience.rb

```diff
@@ -152,6 +153,36 @@ def status
       Cursor.new(nil, request).to_a.select { |change| change[:tailored_audience_id] == id }
     end
 
+    # This real-time API will enable partners to upload batched tailored audience information to Twitter for processing in real-time.
```

Comment:
Can you please add a comment that this is a private API and requires whitelisting from Twitter, then I will approve and check this into the next release